Vagrantfile

```diff
@@ -3,10 +3,11 @@ Vagrant.configure("2") do |config|
     config.vm.box_version = ">= 0.3, < 0.4"
     config.vm.provider "virtualbox" do |vb|
         vb.linked_clone = true
-        vb.memory = 2048
+        vb.memory = 10000
```

Comment:
Do we really need 10000MB memory?
Please consider this amount, and if possible, we should at least keep it under 8GB (8096MB)

Reply:
Not tested at all, was set high just to avoid any problems while developing.